### PR TITLE
#33 Shift Introduction

### LINKED ISSUE #33: Shift Introduction
This issue addresses the content of  Shift's introduction page. This page should provide a short description of what Shift is, key features and why Shift is ideal for the user.  It should also provide some links to the first steps for getting shift started.

### PR DESCRIPTION
**Pull request by juliavisserinbibo**
## Issue
Closes #33

## Descritpion
This issue addresses the content of Shift's introduction page. This page should provide a short description of what Shift is, key features and why Shift is ideal for the user. It should also provide some links to the first steps for getting shift started.

### Note
I based this page on https://inbibo.co.uk/docs/adonisfx/v1.2.0/introduction introduction page and the content from the Shift product page. 
Please, if you want to add anything or reword a section let me know.

### CODE REVIEW

**Review by JorgeHernandezibanezinbibo — APPROVED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -1,15 +1,29 @@
-# Welcome to Shift
+# Introduction
+
+Welcome to the technical documentation of Shift where we provide you with all the information you need to use the software in your digital characters. If you need more information or help, please do not hesitate to send an email to shiftsupport@inbibo.co.uk with your consult.
+
+## Key Values 
 
 Shift is a node-based low-code automation software entirely written in Python.  
-With Shift you can create your own automation [workflows](intro/terminology.md/#workflow), and share them with others.
+With Shift you can use libraries of operators and workflows, connect them together and intuitively build complex processes. Being based on Python, means that Shift gives you the freedom to implement any type of workflow.
```
Comment (leegeertseninbibo) [inline docs/index.md:8]:
```suggestion
With Shift you can use libraries of operators, connect them together and intuitively build workflows to run complex processes. Being based on Python, means that Shift gives you the freedom to implement any type of workflow.
```

**Review by leegeertseninbibo — CHANGES_REQUESTED** [review]

File: docs/index.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
-# Welcome to Shift
+# Introduction
+
+Welcome to the technical documentation of Shift where we provide you with all the information you need to use the software in your digital characters. If you need more information or help, please do not hesitate to send an email to shiftsupport@inbibo.co.uk with your consult.
```
[the remark this answers is not in the document]
Comment (juliavisserinbibo) [reply docs/index.md]: Done

**Review by leegeertseninbibo — APPROVED** [review]

**Review by albertochimentiinbibo — CHANGES_REQUESTED** [review]

**Review by hectorjimenezinbibo — APPROVED** [review]

File: docs/index.md [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
+
+Welcome to the technical documentation of Shift where we provide you with all the information you need to use the software to create workflows to automate tasks. If you need more information or help, please do not hesitate to send an email to shiftsupport@inbibo.co.uk with your consult.
+
+## Key Values 
 
 Shift is a node-based low-code automation software entirely written in Python.  
-With Shift you can create your own automation [workflows](intro/terminology.md/#workflow), and share them with others.
+With Shift you can use libraries of operators, connect them together and intuitively build workflows to run complex processes. Being based on Python, means that Shift gives you the freedom to implement any type of workflow.
+
+- **Easy to Use**: Most automation in VFX, Animation and Video Games relies on the availability of people with software development skills. Shift node-based approach is more familiar to many digital artists and technical directors, allowing everyone to intuitively contribute to the technology stack.
```
[the remark this answers is not in the document]
Comment (juliavisserinbibo) [reply docs/index.md]: Done

File: docs/index.md [hunk 4]
```diff
@@ -1,15 +1,29 @@
...
+- **Lean & Scalable**: Working with many interconnected pipeline scripts makes it difficult to follow the pace enforced by production. Using operator catalogs and working with processes as assets (not software) allows to rapidly adapt and address any production and business requirement.
```
[the remark this answers is not in the document]
Comment (juliavisserinbibo) [reply docs/index.md]: Done

File: docs/index.md [hunk 5]
```diff
@@ -1,15 +1,29 @@
...
+
+## Why Shift?
+
+With Shift you can:
+
+- better understand, configure and scale complex processes,
+- create custom operators for use case specific scenarios,
+- improve efficiency at authoring workflows in your pipeline,
+- have a common technology foundation for your team.
```
[the remark this answers is not in the document]
Comment (juliavisserinbibo) [reply docs/index.md]: Done

**Review by albertochimentiinbibo — APPROVED** [review]

**Pull request merged**